tests/python/gpu/test_gluon_gpu.py

```diff
@@ -647,3 +647,31 @@ def test_gemms_true_fp16():
     assert_almost_equal(ref_results.asnumpy(), results_trueFP16.asnumpy(),
                         atol=atol, rtol=rtol)
 
+@with_seed()
+def test_cudnn_dropout_reproducibility():
```

Comment:
@assert_raises_cudnn_not_satisfied?

Reply:
:+1:

Reply:
Ok, so this does not really work since the test does not really throw when cudnn is not supported. I will revert that.

Reply:
you can use runtime feature detection to determine whether to skip the test.